src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,117 @@
+package duke.ui;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.task.Task;
+/**
```

Comment:
Perhaps there should be a line break here (separating the import statements and top-level class)? [(Reference)](https://google.github.io/styleguide/javaguide.html#s3-source-file-structure)

```suggestion

/**
```